packages/admin-theme/src/CometAdminColorPickerOverrides/colorPicker.ts

```diff
@@ -1,7 +1,7 @@
 import { CometAdminColorPickerClassKeys } from "@comet/admin-color-picker";
 import { StyleRules } from "@material-ui/styles/withStyles";
 
-export const cometAdminColorPickerOverrides = (): StyleRules<{}, CometAdminColorPickerClassKeys> => ({
+export const getCometAdminColorPickerOverrides = (): StyleRules<{}, CometAdminColorPickerClassKeys> => ({
```

Comment:
do we need the CometAdmin prefix on all Overrides or wouldn't it be better to reduce it to e.g.: `getColorPickerOverrides()` or `colorPickerOverrides()` for better readability.

Reply:
I think you're right, including the full component-name in the function-name probably doesn't bring any benefit.